abdiff/core/init_job.py

```diff
@@ -1 +1,36 @@
 """abdiff.core.init_job"""
+
+import logging
+import os
+
+from abdiff.config import Config
+from abdiff.core.utils import (
+    get_job_slug_and_working_directory,
+    update_or_create_job_json,
+)
+
+CONFIG = Config()
+
+logger = logging.getLogger(__name__)
+
+
+def init_job(job_name: str) -> dict:
+    """Function to initialize a new Job.
+
+    1. create a working directory for job
+    2. initialize a job.json file
+    """
+    job_slug, job_working_directory = get_job_slug_and_working_directory(job_name)
```

Comment:
Building on the `output_directory` comment, to further minimize potential confusion, could this just be `job_directory` or `working_directory` consistently?

Reply:
I agree with @ehanson8 on choosing one naming convention and keeping it consistent across functions.
 
I am leaning towards `job_directory`. I like that it is explicitly linking the directory to a job.  Related to [@ehanson8 's comment on `output_directory`](https://github.com/MITLibraries/transmogrifier-ab-diff/pull/2/files#r1731591966): `data_directory` makes me think of a folder containing file extracts and transformed record JSON files, but this directory will contain more than that -- the structure of the directory is very specific to a job, containing job and run JSON files and dated 'run' folders _in addition to_ the transformed records. 🤔